docs/docs-content/vm-management/create-manage-vm/deploy-vm-from-template.md

```diff
@@ -55,6 +55,16 @@ These steps will help guide you to deploy a VM from an out-of-the-box VM templat
 
 8. Give the VM a name and specify memory and CPUs.
 
+   :::info
+
+   You can change the memory and CPU values after the VM is deployed. If you change the memory post-deployment, you can
+   reboot the VM to apply the changes. By default, the Hotplug feature is enabled for both
+   [memory](https://kubevirt.io/user-guide/compute/memory_hotplug/#memory-hotplug-in-action) and
+   [CPU](https://kubevirt.io/user-guide/compute/cpu_hotplug/), allowing changes to be applied without rebooting the VM
+   through live migration.
```

Comment:
```suggestion
   You can change the memory and CPU values after the VM is deployed. By default, the Hotplug feature is enabled for both [memory](https://kubevirt.io/user-guide/compute/memory_hotplug/#memory-hotplug-in-action) and [CPU](https://kubevirt.io/user-guide/compute/cpu_hotplug/), allowing changes to be applied without rebooting the VM through live migration.
```

I removed one sentence to emphasize that it can be done by default.

Reply:
Much cleaner @cloudmaniac . Let's go with your version 👍🏻